src/cfclient_config.erl

```diff
@@ -199,7 +199,12 @@ create_tables(Config) ->
     metrics_cache_table := MetricsCacheTable,
     metrics_counter_table := MetricsCounterTable
   } = Config,
-  ConfigTable = ets:new(ConfigTable, [named_table, set, public, {read_concurrency, true}]),
+  case ets:whereis(ConfigTable) of
```

Comment:
This is the bug fix. Only create global config table shared by each instance once per application.